modules/economy-system/economy-system.js

```diff
@@ -233,8 +233,8 @@ async function leaderboard(client) {
         .setFooter({text: client.strings.footer, iconURL: client.strings.footerImgUrl});
 
     if (model.length !== 0) embed.addField('Leaderboard:', await topTen(model, client));
-    if (moduleStr['leaderboardEmbed']['thumbnail']) embed.setThumbnail(moduleStr['leaderboardEmbed']['thumbnail']);
-    if (moduleStr['leaderboardEmbed']['image']) embed.setImage(moduleStr['leaderboardEmbed']['image']);
+    if (moduleStr['leaderboardEmbed']['thumbnail'] !== ' ') embed.setThumbnail(moduleStr['leaderboardEmbed']['thumbnail']);
+    if (moduleStr['leaderboardEmbed']['image'] !== ' ') embed.setImage(moduleStr['leaderboardEmbed']['image']);
```

Comment:
```suggestion
    if (moduleStr['leaderboardEmbed']['thumbnail'].trim() !== '') embed.setThumbnail(moduleStr['leaderboardEmbed']['thumbnail']);
    if (moduleStr['leaderboardEmbed']['image'].trim() !== '') embed.setImage(moduleStr['leaderboardEmbed']['image']);
```

Comparing against one space doesn't seem to make sense, this makes it even work if users use two spaces! :hAA:

Reply:
I did it, because one space is for some reason the value it has when you don’t add :hAA: But fair point ^^